### DISCUSSION TITLE
Multiple device

### DISCUSSION THREAD

**Discussion by febriansasi** [Freika/dawarich#1104] [Ideas]
Is it possible to tracking multiple device using same username?

**Comment by BeastleeUK** [comment]
I believe the API keys are account specific and  the instructions also show using the email address of the user in the watched folders. I should imagine this means that as long as you apply one of these to whichever data source you're sending to Darwarich.

**Comment by Freika** [comment]
Not yet, but it will be supported

**Comment by Martin22** [comment]
@Freika 
Is it still in the plan? Can you provide a timeframe for when support for multiple devices might be implemented?

**Comment by Freika** [comment]
@Martin22 it's still planned, no estimations so far

**Comment by MexHigh** [comment]
Where would I start implementing this if I'm willing to help? I've read somewhere that the database already contains device IDs which are however not yet reflected in the map. Is this still true?

I want to use this feature to migrate from Traccar, where I track my Car and my Phone's in one account.
